### ISSUE TITLE
As a user with a disability, I should be able to find the buttons to delete an existing flyer using a screen reader.

### ISSUE THREAD

**Issue by dmac1108** [dmac1108/a-flyer-on-the-wall-client#1]
Estimate to complete: 20 minutes
Add appropriate aria labels to button elements
